pytorch_lightning/accelerators/gpu.py

```diff
@@ -16,7 +16,6 @@ def setup(self, trainer, model):
             raise MisconfigurationException(f"Device should be GPU, got {self.root_device} instead")
         self.set_nvidia_flags()
         torch.cuda.set_device(self.root_device)
-        model.to(self.root_device)
         return super().setup(trainer, model)
 
```

Comment:
Just wanted to double check here with @tchaton and @justusschock 
is it good to remove this? any implications for rpc/sequential?
For the  other plugins I have not seen anything breaking.

Reply:
I think this is fine